packages/functions/src/notifications/AppointmentNotificationData.ts

```diff
@@ -27,12 +38,24 @@ export function getAppointmentNotificationData(
     donorName: donor.firstName + " " + donor.lastName,
     donorPhone: donor.phone,
     donationStartTimeMillis: appointment.donationStartTime.toMillis(),
-    unsubscribeLink: getUnsubscribeLink(donor.id),
+    unsubscribeLink: getUnsubscribeLink(isProduction, donor.id),
+    donationApprovalLink: getAppointmentApprovalLink(
+      isProduction,
+      donor.id,
+      appointment.id!,
+      false
+    ),
+    donationNoShowLink: getAppointmentApprovalLink(
+      isProduction,
+      donor.id,
+      appointment.id!,
+      true
+    ),
   };
 }
 
-function getUnsubscribeLink(donorId: string) {
-  if (isProd()) {
+function getUnsubscribeLink(isProduction: boolean, donorId: string) {
+  if (isProduction) {
     return (
       "https://us-central1-blood-components-9ad48.cloudfunctions.net/unsubscribe?method=email&userId=" +
```

Comment:
Maybe it will be best to put the hosts in constants of PROD_HOST_URL and DEV_HOST_URL.